docs/pages/router/migrate/from-expo-webpack.mdx

```diff
@@ -0,0 +1,340 @@
+
+```json package.json
+{
+  "homepage": "/evanbacon/my-website"
+}
+```
+
+In Expo Router, you can use the experimental `basePath` field in the project's **app.json**:
+
+```json app.json
+{
+  "expo": {
+    "experiments": {
+      "basePath": "/evanbacon/my-website"
+    }
+  }
+}
+```
+
+Unlike the previous system, this will also update the routing to account for the base path. For example, if you have a route `/profile` and you set the base path to `/evanbacon/my-website`, then the route will be `/evanbacon/my-website/profile`.
+
+See [hosting with sub-paths](/more/expo-cli#hosting-with-sub-paths) for more information.
+
+## Fast refresh
+
+In `@expo/webpack-config` you could install `@pmmmwh/react-refresh-webpack-plugin` and add the following to the **webpack.config.js**:
+
+```js webpack.config.js
+const createExpoWebpackConfigAsync = require('@expo/webpack-config');
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+
+module.exports = async function (env, argv) {
+  const config = await createExpoWebpackConfigAsync(env, argv);
+
+  // Use the React refresh plugin in development mode
+  if (env.mode === 'development') {
+    config.plugins.push(new ReactRefreshWebpackPlugin({ disableRefreshCheck: true }));
+  }
+
+  return config;
+};
+```
+
+In Expo Router, **Fast Refresh is enabled by default** using the official Fast Refresh implementation by Meta.
+
+## Favicons
+
+Like `@expo/webpack-config`, Expo Router supports generating the **favicon.ico** file based on the `web.favicon` field in the **app.json**.
+
+## Service workers
+
+> **warning:** Be careful adding service workers as they are known to cause unexpected behavior on web. If you accidentally ship a service worker who aggressively caches your website, users cannot request updates easily. For the best offline mobile experience, create a native app with Expo. Unlike websites with service workers, native apps can be updated through the app store to clear the cached experience. This would be similar to resetting the user's native browser (which they may have to do if the service worker is aggressive enough). See [why service workers are suboptimal](https://github.com/facebook/create-react-app/issues/2398) for more information.
+
+Expo Webpack didn't have built-in service worker support. However, you could add it yourself by using the `workbox-webpack-plugin` and adding it to the **webpack.config.js**.
+
+Workbox doesn't have a Metro integration, but because Workbox doesn't require one of the core features of a bundler (transformation, resolution, serialization), it can easily be used as a post-build step. Follow the guide for [using Workbox CLI](https://developer.chrome.com/docs/workbox/modules/workbox-cli/), and wherever it refers to a "build script" use `npx expo export -p web` instead.
+
+For example, here's a possible flow for setting up Workbox. Create a new project with the following command:
+
+<Terminal cmd={
```

Comment:
```suggestion
<Terminal cmd={[
```

This was missing from the changes I suggested earlier and will help fix the CI issue.